com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -792,3 +792,4 @@ max_time_step.default = 0.033333
 runtime_generation.type = bool
 runtime_generation.help = Use runtime font generation
 runtime_generation.default = 0
+runtime_generation.private = 1
```

Comment:
It seems like we don't use this parameter in runtime, so we don't need it in the final `game.projectc` in bundle